src/components/DialogBody.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import CloseIcon from './CloseIcon';
+
+const DialogBody = ({ locationDetails, onClick }) => (
+  <>
+    <button
+      className="dialog__button"
+      aria-label="close dialog"
+      onClick={onClick}
+    >
+      <CloseIcon />
+    </button>
+
+    <div className="dialog__image">
+      <img
+        src={
+          locationDetails?.thumbnail?.source ||
+          `${process.env.PUBLIC_URL}/placeholder.png`
+        }
+        alt={locationDetails?.thumbnail?.source ? locationDetails.title : ''}
+      />
+    </div>
+    <div className="dialog__body">
+      <a
+        className="dialog__body__link"
+        href={`https://en.wikipedia.org/?curid=${locationDetails.pageid}`}
+      >
+        <h2>{locationDetails.title}</h2>
+      </a>
+      <p>{locationDetails.description}</p>
+      <p>Distance: {locationDetails.coordinates[0].dist} km</p>
+      <p></p>
```

Comment:
This extra `<p></p>`, I couldn't identify the purpose, but if you want to create some space. I suggest using `<br />` or `margin-bottom`.

Reply:
I think it's a leftover from our refactoring 😄, going to remove it now, thank you so much for pointing it out